goal_src/jak2/levels/city/common/vehicle-physics.gc

```diff
@@ -6,814 +6,3 @@
 ;; dgos: CWI
 
 ;; DECOMP BEGINS
-
-(defmethod vehicle-method-99 vehicle ((obj vehicle) (arg0 float))
-  (let ((s4-0 (new 'stack-no-clear 'vehicle-grab-rail-info))
```

Comment:
had to revert this file, one of the methods isn't correct -- it uses `rigid-body-vehicle-constants` in a strange way that wasn't properly figured out